hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -130,6 +135,14 @@ public SCMNodeManager(OzoneConfiguration conf,
     this.useHostname = conf.getBoolean(
         DFSConfigKeysLegacy.DFS_DATANODE_USE_DN_HOSTNAME,
         DFSConfigKeysLegacy.DFS_DATANODE_USE_DN_HOSTNAME_DEFAULT);
+    this.scmhaManager = scmhaManager;
+  }
+
+  public SCMNodeManager(OzoneConfiguration conf,
```

Comment:
Any case SCMNodeManager would not require SCMHAManager?
Note that PipelineManager will always rely on SCMHAManager

Reply:
Quite a lot of unit tests rely on this old Ctor.